shard-operator/src/main/resources/templates/bridge-executor-deployment.yaml

```diff
@@ -36,7 +36,9 @@ spec:
               value: ""
             - name: KAFKA_GROUP_ID
               value: ""
-            - name: PROCESSOR_CONFIGURATION
+            - name: PROCESSOR_DEFINITION
```

Comment:
@r00ta Is the `PROCESSOR_CONFIGURATION` --> `PROCESSOR_DEFINITION` change intended?

Reply:
@robpblake it was a leftover. The template is just defining some empty env variables that are then replaced programmatically. The correct env var name is `PROCESSOR_DEFINITION`. Everything was running fine, it's just something I noticed and quickly fixed